src/test/java/org/apache/commons/cli/PatternOptionBuilderTest.java

```diff
@@ -159,13 +161,15 @@ public void testURLPattern() throws Exception
     @Test
     public void testExistingFilePattern() throws Exception
     {
-        final Options options = PatternOptionBuilder.parsePattern("f<");
+        final Options options = PatternOptionBuilder.parsePattern("f<g<");
```

Comment:
Looking at this, I think we're doing to many things at once in this test. Why don't we split this test up into to separate tests? One for `f<` and one for `g<`?